warehouse/migrations/versions/42f0409bb702_add_two_factor_and_update_users.py

```diff
@@ -0,0 +1,76 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+add two_factor and update users
+
+Revision ID: 42f0409bb702
+Revises: c0c0544354e7
+Create Date: 2019-03-28 21:01:52.980989
+"""
+
+import sqlalchemy as sa
+
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+revision = "42f0409bb702"
+down_revision = "c0c0544354e7"
+
+# Note: It is VERY important to ensure that a migration does not lock for a
+#       long period of time and to ensure that each individual migration does
+#       not break compatibility with the *previous* version of the code base.
+#       This is because the migrations will be ran automatically as part of the
+#       deployment process, but while the previous version of the code is still
+#       up and running. Thus backwards incompatible changes must be broken up
+#       over multiple migrations inside of multiple pull requests in order to
+#       phase them in over multiple deploys.
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
```

Comment:
Is this table correct? I don't see the unqiue constraint being added. It looks like you can have multiple rows all referring to the same user, but your model above has a unqiue constraint. Did you forget to update the migration after making a code change?

Reply:
Yeah, I think I dropped the migration changes while merging the DB changes last week. Thanks!